ddl/backfilling.go

```diff
@@ -555,11 +543,17 @@ func getBatchTasks(t table.Table, reorgInfo *reorgInfo, kvRanges []kv.KeyRange,
 	//nolint:forcetypeassert
 	phyTbl := t.(table.PhysicalTable)
 	jobCtx := reorgInfo.d.jobContext(job.ID)
-	for i, keyRange := range kvRanges {
+	for _, keyRange := range kvRanges {
 		taskID := taskIDAlloc.alloc()
 		startKey := keyRange.StartKey
+		if len(startKey) == 0 {
+			startKey = prefix
+		}
 		endKey := keyRange.EndKey
-		endK, err := getRangeEndKey(jobCtx, reorgInfo.d.store, job.Priority, prefix, keyRange.StartKey, endKey)
+		if len(endKey) == 0 {
+			endKey = prefix.PrefixNext()
```

Comment:
I just found that index ID is not included :(

`t.IndexPrefix()` is the key like `t[tableID]_i`. If we use `t.IndexPrefix().PrefixNext()` here, `endKey` will be `t[tableID]_j`?

As a result, we will scan the index record from **all** indexes.